languages/go/exercises/concept/booleans/.docs/hints.md

```diff
@@ -0,0 +1,21 @@
+## General
+
+- There are three [boolean operators][logical operators] to work with boolean values.
+- Multiple operators can be combined in a single expression.
+
+## 1. Check if a fast attack can be made
+
+- The logical NOT operator (`!`) can be placed before an expression to negate its value.
+
+## 2. Check if a spy action can be made
+
+- Logical operators apply to boolean values and yield a result of the same type as the operands. The right operand is evaluated conditionally.
```

Comment:
ok, I give up... this is too complicated for me 😄 

Maybe make this hint easier to understand? As we show this when a student is stuck we should view this almost like a little guide of what the student has to do in order to solve the exercise.

Reply:
:)) I've rephrase.